scalafix-rules/src/main/scala/scala/meta/internal/pc/ScalafixGlobal.scala

```diff
@@ -177,8 +177,11 @@ class ScalafixGlobal(
     tpe match {
       case TypeRef(pre, sym, args)
           if Identifier.needsBacktick(sym.decodedName) &&
-            sym != definitions.ByNameParamClass // `=> T` is OK
-          =>
+            sym != definitions.ByNameParamClass && // `=> T` is OK
+            !sym.isPackageObject &&
+            !definitions.isRepeated(sym) &&
+            !sym.isAbstractType &&
```

Comment:
it looks like only this new condition is tested - is that what you mean by "Some of that might not be needed for Scalafix, but it's better to be on the safe side" ?

Reply:
Repeated seems only to happen in in signatures of methods, which is not reproducible in here, so maybe I can remove it?

Reply:
`sym.isPackageObject` was needed for Iterable, but it doesn't seem to break here, just in metals